src/fields.js

```diff
@@ -5,14 +5,34 @@
 // :Param hidden: true -> hide the field, false or default  undefined-> show the field
 
 export const fields = [
-  { label: "All text fields", field: "searchText", type: "text" },
-  { label: "Identifier", field: "id", type: "text" },
-  { label: "Source", field: "source", type: "list-facet", facetSort: "index", collapse: true },
+  { label: "Curation accessContraints", field: "curation_accessContraints", type: "non-search", hidden: true },
```

Comment:
Is the label actually used anywhere?  Does it show up in the exported csv?  If not, could we eliminate it or default to the field?

Reply:
To default it to the field might be a good idea. I will make a change.